doc/classes/CollisionObject2D.xml

```diff
@@ -16,7 +16,7 @@
 			<param index="1" name="event" type="InputEvent" />
 			<param index="2" name="shape_idx" type="int" />
 			<description>
-				Accepts unhandled [InputEvent]s. [param shape_idx] is the child index of the clicked [Shape2D]. Connect to the [code]input_event[/code] signal to easily pick up these events.
+				Accepts unhandled [InputEvent]s. [param shape_idx] is the child index of the clicked [Shape2D]. Connect to [signal input_event] to easily pick up these events.
```

Comment:
Random thought, but since this name might be ambiguous, I understand why one might want to make "signal" explicit in the text so it's not confused to methods like `_input_event` in this same class.

I'm not advocating to change it back, but it makes me think that we should probably aim to have a tooltip on all those hyperlinks providing context on the type of API, and maybe its short description.

This would prevent having to click a link to know what it's referring to exactly, before going back in history where one was.

Totally for a separate PR / proposal, and after Yeldham's refactor is merged.

Reply:
> I'm not advocating to change it back, but it makes me think that we should probably aim to have a tooltip on all those hyperlinks providing context on the type of API, and maybe its short description.

I was thinking an icon, but we should also be able to add tooltips now, yeah.